src/app/site/home/slide-show/slide-show.component.ts

```diff
@@ -37,4 +41,10 @@ export class SlideShowComponent implements OnInit {
       });
 
   }
+  setPosition(positionStr, containerStr) {
+    this.scrollService.position = positionStr;
+    if (this.router.url === '/about-us') {
+      this.scrollService.triggerScrollTo();
+    }
```

Comment:
no need to check this ocndition, it may occure when we are in about-us page and click on footer items only